java/opendcs/src/main/resources/db/CWMS-Oracle/schema/V7.2__multi_office.sql

```diff
@@ -0,0 +1,47 @@
+-- CCP Schema Update
+-- June 2025
+-- Purpose :  More support for Multi Office, some size increases
+-- Strategy:
+--          - add db_office_code as needed
+--          - db_office_code often added to existing unique constraints
+
+
+ 
+ -- increase description size for platformconfig table; see: https://github.com/opendcs/opendcs/issues/1154
+ALTER TABLE CCP.PLATFORMCONFIG MODIFY (DESCRIPTION VARCHAR2(4000)); 
+ -- , Error Message = ORA-12899: value too large for column "CCP"."DATASOURCE"."DATASOURCEARG" (actual: 413, maximum: 400)
+ALTER TABLE CCP.DATASOURCE MODIFY (DATASOURCEARG VARCHAR(600));
```

Comment:
This should also be 4000.
I think Jessica has a few URLs that would trip even 600 over the edge.